TASK/ClearPedTasks.md

```diff
@@ -9,6 +9,7 @@ void CLEAR_PED_TASKS(Ped ped);
 ```
 
 Clear a ped's tasks. Stop animations and other tasks created by scripts.
+If a player uses this native, on another player's ped, this native triggers clearPedTasksEvent server event, if onesync is enabled.
```

Comment:
Same for this.

> It will trigger the event regardless of onesync being enabled or not, especially if the net object (ped) is remote. It will send the event from your client and the server will route it to others.